suites/suite_zhang_mcfarlane.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="zhang_mcfarlane" version="1.0">
+   <group name="physics_before_coupler">
+       <scheme>initialize_constituents</scheme>
+       <scheme>to_be_ccppized_temporary</scheme>
+       <scheme>zm_convr</scheme>
```

Comment:
I'm not sure if this is within the scope of the PR, but maybe the calls to ZM need to be added to the corresponding locations in the CAM7 SDF as well.

Reply:
Added ZM to the CAM7 SDF

Reply:
Thanks @cacraigucar !